compiler/src/iree/compiler/Codegen/ExternalInterfaces/CPUEncodingExternalModels.cpp

```diff
@@ -689,7 +662,8 @@ enumerateVMVXMatmulTiles(linalg::ContractionDimensions cDims,
   // codegen.query_tile_sizes op, so we disable dynamic tile shapes for
   // batch_matmul. Also, they are not set up for narrow M/N matmul, so it is
   // disabled when it is the case.
-  if (!cDims.batch.empty() || getMatmulNarrowDim(encoding)) {
+  if (!cDims.batch.empty() ||
+      getMatmulNarrowDim(encoding, /*powerOf2Ceil*/ true)) {
```

Comment:
style nit: https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments

```suggestion
      getMatmulNarrowDim(encoding, /*powerOf2Ceil=*/true)) {
```

Reply:
Not applicable anymore after renaming to `getPo2MatmulNarrowDim`